Resources/config/schema/seo-1.0.xsd

```diff
@@ -39,4 +39,8 @@
             <xsd:enumeration value="canonical" />
         </xsd:restriction>
     </xsd:simpleType>
+
+    <xsd:simpleType name="enable_content_listener">
```

Comment:
these changes should be removed and instead, you should add this between line 20 and 21 in this file:

````xsd
<xsd:attribute name="enable-content-listener" type="xsd:boolean" default="true" />
````